src/core/shaderlib/project/viewport-uniforms.js

```diff
@@ -187,8 +180,10 @@ export function getUniformsFromViewport({
 
     // Distance at which screen pixels are projected
     project_uFocalDistance: viewport.focalDistance || 1,
-    project_uPixelsPerUnit: distanceScales.pixelsPerMeter,
+    project_uPixelsPerMeter: distanceScales.pixelsPerMeter,
     project_uPixelsPerDegree: distanceScales.pixelsPerDegree,
+    project_uPixelsPerUnit: distanceScales.pixelsPerMeter,
```

Comment:
Yes, this makes sense. Meters are for distances, and units for positions. 

I feel that we could make it even clearer by changing `project_uPixelsPerUnit` to `project_uPixelsPerPositionUnit`.

On the downside, the number of `project` uniforms keeps growing...